backend/src/shortener/shortener.service.ts

```diff
@@ -1,9 +1,13 @@
 import { AppCacheService } from './../cache/cache.service';
 import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
 
 @Injectable()
 export class ShortenerService {
-  constructor(private readonly appCacheService: AppCacheService) {}
+  constructor(
+    private readonly appCacheService: AppCacheService,
+    private readonly configService: ConfigService
```

Comment:
For type safety, use the config wrapper instead of the normal ConfigService. Use AppConfigService.